save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/basic/TimelineComponent.kt

```diff
@@ -0,0 +1,60 @@
+@file:Suppress("FILE_NAME_INCORRECT", "FILE_NAME_MATCH_CLASS")
+
+package com.saveourtool.save.frontend.components.basic
+
+import react.*
+import react.dom.html.ReactHTML.div
+import web.cssom.*
+
+import kotlinx.datetime.LocalDateTime
+
+val timelineComponent: FC<TimelineComponentProps> = FC { props ->
+    div {
+        className = ClassName("mb-3")
+        div {
+            className = ClassName("mt-3 mb-3 text-xs text-center font-weight-bold text-primary text-uppercase")
+            +"Timeline"
+        }
+        div {
+            className = ClassName("card card-body p-0 timeline-container")
+            div {
+                className = ClassName("steps-container")
+                div {
+                    className = ClassName("line completed")
+                }
+                props.dates
+                    .map { (dateTime, label) -> dateTime.date to label }
+                    .sortedBy { it.first }
```

Comment:
suggestion to sort by dateTime, just for case when two actions happen at the same day

Reply:
ok, will do it

Reply:
Done